### ISSUE TITLE
re-factor

### ISSUE THREAD

**Issue by oravirt** [oravirt/ansible-oracle#71]
I haven't really done a lot to improve the old code up until now, so I'm going through that and adjusting it where it is needed. This will probably be a an on-going thing for a while.
I will try not to change variable definitions if I don't have to, to minimize the potential impact.

**In general**:
Up until now, I haven't really used any v2.x specific things but that will change as of now.

`oraswdb-install` will be changed to use loop_control to iterate over each home. When this change is pushed Ansible >= 2.1 is needed

**New roles** (some will not work with 1.x):

`orahost-ssh` - will replace the old way of dealing with ssh-keys in RAC (already available)
`oradb-manage-db` - will consolidate (and replace) oradb-create & oradb-delete

The following roles will also be coming shortly.
```
oradb-manage-pdb
oradb-manage-users
oradb-manage-roles
oradb-manage-grants
oradb-manage-tablespaces
oradb-manage-services
oradb-manage-parameter
```

**Comment by oravirt** [comment]
orahost-ssh - done as of b01cf5238a7c9b15ca371199f46f8cab6569bd80
oradb-manage-db - done as of 87341e870c54e121fd8fec05d744499c1ee5c227

**Comment by oravirt** [comment]
`oraswdb-install `- done as of 73117d909fbe97a0c2987a16d5ab7b7771b2d6e4 (not merged to master yet, changes are in oraswdb-loop-control branch)
This change will require `Ansible version >= 2.1`

**Comment by oravirt** [comment]
`oraswdb-loop-control` is merged to `master` as of 4b1bc895389d7ec2553614064b1514e91e080496
